src/client/App.js

```diff
@@ -26,6 +27,9 @@ function App() {
         <Route exact path="/">
           <LandingPageContainer />
         </Route>
+        <Route exact path="/Container/ProductPage/:id">
```

Comment:
Pls change this route to `product/:id`, that is more logical

Reply:
done

Reply:
its better to use 'products/:id' .. same like api/products/:id

Reply:
It is looks ok ...api/product/:id